src/constants/index.tsx

```diff
@@ -0,0 +1 @@
+export const API_URL = "http://127.0.0.1:8000/api/students/"
```

Comment:
Best practice to expose these as environment variables with a default, e.g
```suggestion
export const API_URL = process.env.REACT_APP_API_URL || "http://127.0.0.1:8000/api/students/"
```